### PR TITLE
Added the ability to specify ovf properties during deployment

### LINKED ISSUE #288: govc import.ova doesn't provide vApp properties
govc import.ova doesn't provide the way to specify vApp properties for VM.

### PR DESCRIPTION
**Pull request by bruceadowns**
* added import.spec target to produce example json
* added -options argument to import.ovf/import.ova targets
* moved -folder argument flag class
* updated change log
* minor formatting changes
* closes #288

### CODE REVIEW

File: govc/importx/spec.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	o := flags.Options{
+		deploymentOptions,
+		deploymentOptions[0],
+		allDiskProvisioningOptions,
+		allDiskProvisioningOptions[0],
+		allIpAllocationPolicyOptions,
+		allIpAllocationPolicyOptions[0],
+		allIpProtocolOptions,
+		allIpProtocolOptions[0],
+		cmd.Map(e)}
```
Comment (pietern) [inline govc/importx/spec.go]: Please call out the fields that you're assigning to. Extending the options struct is cumbersome when it is initialized like this.

**Comment by pietern** [comment]
Can you move the import flag to the importx package, since it only relevant for OVF importing.

The flags in the flags package are typically used across commands in multiple packages.

File: govc/importx/spec.go (hunk 1 above)
Comment (bruceadowns) [inline govc/importx/spec.go]: Like this?

    o := flags.Options{
        AllDeploymentOptions: deploymentOptions,
        DeploymentOption: deploymentOptions[0],
        AllDiskProvisioningOptions: allDiskProvisioningOptions,
        DiskProvisioning: allDiskProvisioningOptions[0],
...
`

I had actually already changed this in my current change set. :-)

Comment (pietern) [inline govc/importx/spec.go]: Yeah, that's what I meant :)

**Comment by dougm** [comment]
Nice, LGTM

**Comment by bruceadowns** [comment]
Thanks for checking it out, pieter and doug.

**Comment by pietern** [comment]
Thanks Bruce, merging this PR.

**Pull request merged**